### PR TITLE
Add rake task to purge compact index from Fastly

### PR DESCRIPTION
**Pull request by indirect**
I already did the Fastly configuration to make the wildcard purge work, but the wildcard requires that we do one last purge all before the surrogate keys will be tagged correctly so we can purge them together.

/cc @segiddins @dwradcliffe

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -222,3 +222,11 @@ task :yank_spec, :name, :version, :platform do |t, args|
   end
   puts "Yanked #{version}!"
 end
+
+desc "Purge new index from Fastly cache"
+task :purge_cdn do
+  cdn = BundlerApi::CacheInvalidator.new.cdn_client
+  cdn.purge_key("names")
+  cdn.purge_key("versions")
```
Comment (segiddins) [inline Rakefile]: These can just purge the path?

**Comment by dwradcliffe** [comment]
Where are the surrogate keys configured?

**Comment by indirect** [comment]
Content → Headers → `Default Wildcard` / `info/* Wildcard`

File: Rakefile (hunk 1 above)
Comment (indirect) [inline Rakefile]: Yeah, that's true. Alright...

**Comment by segiddins** [comment]
@homu r+

**Pull request merged**